src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -685,22 +685,16 @@ public function testRestart()
      */
     public function testRunProcessWithTimeout()
     {
-        $timeout = 0.1;
-        $process = $this->getProcess(self::$phpBin.' -r "sleep(1);"');
-        $process->setTimeout($timeout);
+        $process = $this->getProcess(self::$phpBin.' -r "sleep(30);"');
```

Comment:
this fixes [a transient test](https://travis-ci.org/symfony/symfony/jobs/100528238#L4657)